### PR TITLE
To support new device SNH60A0_640F.

### PR DESCRIPTION
**Pull request by Mingxikao**
Alphanetworks SNH60A0-640F : add to support ONIE. 
The machine has 64 x 100 Gigabit Ethernet QSFP28 ports with Broadcom BCM56970 switch ASIC. 
The CPU is Intel Rangeley C2558 Quad Cores/2.4GHz.

### CODE REVIEW

File: machine/alphanetworks/alphanetworks_snh60a0_640f/machine.make [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+VENDOR_VERSION = .0.1
+
+# Vendor ID -- IANA Private Enterprise Number:
+# http://www.iana.org/assignments/enterprise-numbers
+# Alpha Networks Inc.
+VENDOR_ID = 31874
+
+# Enable the i2ctools and the onie-syseeprom command for this platform
+I2CTOOLS_ENABLE = yes
+I2CTOOLS_SYSEEPROM = no
```
Comment (cbrune) [inline machine/alphanetworks/alphanetworks_snh60a0_640f/machine.make]: As of commit e633c568746317624c068e96c706ee55deaef9eb `I2CTOOLS_SYSEEPROM = no` is the default.  No need to explicitly set it anymore

File: machine/alphanetworks/alphanetworks_snh60a0_640f/kernel/drivers-base-dd-alpha-snh60a0-640f.patch [hunk 2]
```diff
@@ -0,0 +1,24 @@
+diff --git a/drivers/base/dd.c b/drivers/base/dd.c
```
Comment (cbrune) [inline machine/alphanetworks/alphanetworks_snh60a0_640f/kernel/drivers-base-dd-alpha-snh60a0-640f.patch]: Please do not include the same patch, over and over for each machine.

You can share patches across machines by putting them in a top level `machine/<vendor>/kernel` directory.

For example see d43c0007c0ad5e901ae1f0d960b5f1fb0bacb923 which adds a common kernel patch used by multiple accton machine.

**Comment by cbrune** [comment]
All in all this seems fine.  I suggested some cleanups to help you better maintain the code.

Also please make the commit message more informative.  For example see 121cc7da3bb0214d64ad30a0464b923a15fe33d7.

**Comment by Mingxikao** [comment]
Thanks for your comments. I had added more detail in the comment and also removed the unnecessary definition.  I am planing to merge the kernel patch for all related machines in next commitment.

**Pull request closed without merging**